src/ui/public/errors.js

```diff
@@ -21,272 +19,254 @@ function KbnError(msg, constructor) {
     this.stack = '';
   }
 }
-errors.KbnError = KbnError;
 _.class(KbnError).inherits(Error);
 
 /**
  * If the error permits, propagate the error to be rendered on screen
- * @param handler the handlers that can render the error message to the screen.
  */
 KbnError.prototype.displayToScreen = function () {
   throw this;
 };
 
 /**
  * HastyRefresh error class
- * @param {String} [msg] - An error message that will probably end up in a log.
  */
-errors.HastyRefresh = function HastyRefresh() {
+export function HastyRefresh() {
   KbnError.call(this,
     'Courier attempted to start a query before the previous had finished.',
-    errors.HastyRefresh);
-};
-_.class(errors.HastyRefresh).inherits(KbnError);
```

Comment:
Not _really_ relevant to this PR, but as we're changing this code I'll ask:

Any reason we can't have `class KbnError extends Errors` and `class HastyRefresh extends KbnError`? If that works it would clean up this code even more.